scripts/spack/packages/serac/package.py

```diff
@@ -110,6 +110,10 @@ class Serac(CachedCMakePackage, CudaPackage):
     depends_on("camp@0.1.0serac", when="+raja")
     depends_on("raja@0.13.1serac~openmp~shared", when="+raja")
     depends_on("umpire@5.0.1~shared", when="+umpire")
+    # Lump in CHAI with Umpire for now
+    depends_on("chai@2.3.1serac~shared", when="+umpire")
+    # Our RAJA version is too new for CHAI
```

Comment:
That's unfortunate

Reply:
This may actually not be true anymore as I think there has been a round of releases since I put that comment in